ariadne/cardano/test/backend/Test/Spec/Keystore.hs

```diff
@@ -4,12 +4,12 @@ module Test.Spec.Keystore (
     spec
   ) where
 
-import System.Directory (doesFileExist, getTemporaryDirectory, removeFile)
-import System.FilePath ((</>))
+import System.Directory (doesFileExist, removeFile)
 import System.IO.Error (IOError)
 
 import Test.Hspec (Spec, describe, it, shouldBe, shouldReturn)
 import Test.Hspec.QuickCheck (prop)
+import Test.Pos.Core.Arbitrary ()
```

Comment:
Why do you need this import?

Reply:
I was getting errors related to lack of  class instances from `Test.Pos.Core.Arbitrary () `.

Reply:
Strange that they were not required before.